### PR TITLE
Yaz 52 fix register component

### PR DESCRIPTION
**Pull request by alejandroVaron**
What: Fix register component

Why: Is required to do work the application

Ticket: YAZ-52

### CODE REVIEW

File: front-end-yaz/src/App.vue [hunk 1]
```diff
@@ -173,10 +171,11 @@ export default {
 #name_worker {
   margin-top: 30px;
   margin-bottom: 15px;
-  font-size:x-large;
+  font-size:0.73vw;
 }
 #email_worker {
-  font-size:x-large;
+
```
Comment (juanDuqueS) [inline front-end-yaz/src/App.vue:177]: Format document 👀

File: front-end-yaz/src/components/Login.vue [hunk 2]
```diff
@@ -138,18 +146,23 @@ export default {
               user_address: response.data.data.user_address,
             });
             this.setHideMenu(true);
+            this.overlayError = false;
             this.$router.go(-1);
           } else {
+
```
Comment (juanDuqueS) [inline front-end-yaz/src/components/Login.vue:152]: format document 👀

**Review by juanDuqueS — APPROVED** [review]
There are some blank spaces but does not affect

File: front-end-yaz/src/App.vue (hunk 1 above)
Comment (alejandroVaron) [reply front-end-yaz/src/App.vue]: Thanks, I already fixed it

File: front-end-yaz/src/components/Login.vue (hunk 2 above)
Comment (alejandroVaron) [reply front-end-yaz/src/components/Login.vue]: Thanks, I already fixed it

**Review by prometteo14 — APPROVED** [review]

**Pull request merged**